apps/betterangels-backend/accounts/schema.py

```diff
@@ -181,39 +208,6 @@ def create_client_profile(self, info: Info, data: CreateClientProfileInput) -> C
                 },
             )
 
-            if contacts_data:
```

Comment:
turns out strawberry is smart enough to create the related objects. I don't remember exactly when/why we adopted this pattern (maybe when we had related objects with custom logic and couldn't use the resolver directly) but it's not necessary for create.

Reply:
😮